src/renderer/coremods/installer/index.tsx

```diff
@@ -108,18 +123,28 @@ async function injectLinks(): Promise<void> {
     return;
   }
 
-  injector.before(exports, key, ([args]) => {
-    const { href } = args;
-    if (!href) return undefined;
+  injector.instead(exports, key, (args, fn) => {
```

Comment:
Editing the message doesn't update the embed https://gyazo.com/76026f51ab588bbb7ea7ae71a0442a99
Maybe we can also remove the classic link embed?